_posts/2021-08-18-release-0.10.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Release of a Pure-Rust v0.10 and a Call For Testing
+---
+
+gfx-rs community's goal is to make graphics programming in Rust easy, fast, and reliable. Our main projects are:
+
+  - [wgpu](https://github.com/gfx-rs/wgpu) is built on top of wgpu-hal and naga. It provides safety, accessibility, and portability for graphics applications.
+  - [naga](https://github.com/gfx-rs/naga) translates shader programs between languages, including WGSL. It also provides shader validation and transformation, ensuring user code running on the GPU is safe and efficient.
+
+If you've been following these releases you'll notice that gfx-hal is absent from this list. gfx-hal has now been deprecated in favor of a new abstraction layer inside of `wgpu` called `wgpu-hal`. To see more information about the deprecation, see the [0.9 release]({% post_url 2021-07-16-release-0.9-future %}) post.
+
+Following our release cadence every few months, we rolled out 0.10 through all of the gfx-rs projects! See [wgpu v0.10 changelog](https://github.com/gfx-rs/wgpu/blob/master/CHANGELOG.md#v010-2021-08-18) and [naga v0.6 changelog](https://github.com/gfx-rs/naga/blob/master/CHANGELOG.md#v06-2021-08-18) for the details.
+
+## Pure-Rust Graphics
+
+`wgpu` has had many new changes, the most notible of which is the switch to our new Harware Abstraction Layer `wgpu-hal`. This includes completely rebuilt backends which are more efficient, easier to maintain, and signifigantly leaner. As part of this, we have shedded our last C/C++ dependency `spirv-cross`. We now are entirely based on `naga` for all of our shader translation. This is not only a marked achievement for rust graphics, but has made wgpu safer and more robust.
+
+The new `wgpu-hal`:
+ - Supports Vulkan, D3D12, Metal, and OpenGL ES with D3D11 to come soon.
+ - Has 40% of the code of gfx-hal (22k LOC vs 55k)
```

Comment:
I wonder if it would be more impressive to say "60% less code"?

Reply:
Very likely